website/static/js/citationGrid.js

```diff
@@ -378,10 +368,12 @@ CitationGrid.prototype.makeBibliography = function(folder, format) {
 };
 
 CitationGrid.prototype.getBibliography = function(folder, format) {
-    if (format) {
```

Comment:
Do you know what `format` was used for? It looks like it's not being used with these changes

Reply:
It's redundant in this function, if someone changes the format we remake the citations anyway.

Reply:
Can we remove it as an argument then?

Reply:
👍